spark-commons/src/main/scala/za/co/absa/spark/commons/errorhandling/ErrorHandling.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spark.commons.errorhandling
+
+import org.apache.spark.sql.{Column, DataFrame, SparkSession}
+import za.co.absa.spark.commons.errorhandling.implementations.submits.{ErrorMessageSubmitOnColumn, ErrorMessageSubmitWithoutColumn}
+import za.co.absa.spark.commons.errorhandling.types._
+
+trait ErrorHandling {
+  def putErrorToColumn(errorMessageSubmit: ErrorMessageSubmit): ErrorColumn
+
+  def aggregateErrorColumns(dataFrame: DataFrame)(errCols: ErrorColumn*): DataFrame
```

Comment:
here and everywhere - it might be worth considering to rename errCol / errCols => errSrcCol or srcColWithError or something like that :D

Reply:
Will be part of #101